scripts/bike_philly_data_proc.py

```diff
@@ -0,0 +1,277 @@
+#!/usr/bin/env python
```

Comment:
I assume that /bike_philly_data_proc.py vs .ipynb are the same.  How about just keep one, and clean that one up?